app/index.html

```diff
@@ -11,6 +11,7 @@
 
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/idea-bakers.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
```

Comment:
I agree but remember - normally in Ember we cannot add any line to index.html and for ex. for use FontAwesome you should use external package, like https://github.com/martndemus/ember-font-awesome